castle/cms/browser/controlpanel/security.py

```diff
@@ -2,14 +2,29 @@
 from castle.cms.widgets import SelectFieldWidget
 from plone.app.registry.browser import controlpanel
 from Products.CMFPlone.controlpanel.browser import security
-
+from z3c.form.field import Fields
+from zope.schema import Int
+from plone import api
 
 class SecurityControlPanelForm(security.SecurityControlPanelForm):
     schema = ISecuritySchema
 
     def updateFields(self):
         super(SecurityControlPanelForm, self).updateFields()
         self.fields['restrict_logins_to_countries'].widgetFactory = SelectFieldWidget
+        timeout = api.portal.get().acl_users.session.timeout
```

Comment:
Clever!

Reply:
Yeah, this is cool. Thanks for showing me new tricks too!